.github/workflows/CD.yml

```diff
@@ -122,7 +130,7 @@ jobs:
           echo ${{ steps.set-matrix.outputs.ec2-matrix-2 }}
           echo ${{ steps.set-matrix.outputs.ec2-matrix-3 }}
           
-  release-to-s3:
+  release-version-preparation:
```

Comment:
At first I thought it would be better if we support pull image arm, amd64 on docker for ec2 test, the same as the CI workflow. Therefore, I combined both of these to in case of that since they are already supported on there thought. However, it would be best to do it in another PR or when ever we needs it though (YAGNI).